fulp_modules/main_features/bloodsuckers/code/bloodsucker/bloodsucker_objectives.dm

```diff
@@ -85,33 +85,20 @@
 	switch(rand(0,2))
 		if(0)
 			target_role = "HEAD"
+			target_amount = 1 // Head objectives get one target
 		else
-			target_role = pick(departs)
-	// Now, did we land on Head? We only get one target, then
-	if(target_role == "HEAD")
-		target_amount = 1
-	else
-		// Otherwise, we get 2-3 targets & pick a random Command member, their department is our target.
-		switch(target_role)
-			if("Head of Security")
-				department_string = "Security"
-			if("Head of Personnel")
-				department_string = "Cargo"
-			if("Research Director")
-				department_string = "Science"
-			if("Chief Engineer")
-				department_string = "Engineering"
-			if("Chief Medical Officer")
-				department_string = "Medical"
-		target_amount = rand(2,3)
+			target_role = pick(heads_w_depart_index)
+			department_string = heads_w_depart_index.Find(target_role) // The department is the index!
+			target_amount = rand(2, 3) // Non-head objectives get multiple
 	..()
 
 // EXPLANATION
 /datum/objective/bloodsucker/protege/update_explanation_text()
+	. = ..()
 	if(target_role == "HEAD")
-		explanation_text = "Guarantee a Vassal ends up as a Department Head or in a Leadership role via the Persuasion Rack."
+		explanation_text = "Guarantee a Vassal ends up as a Department Head or in a Leadership role, or convert an existing Head via the Persuasion Rack."
```

Comment:
What's this change for?

Reply:
The wording makes it seem like the vassal is becoming a head or a leadership role as a result of the persuasion rack, at least to me. I can revert if you'd like

Reply:
If you want to do that, this might be better

```suggestion
		explanation_text = "Guarantee a Department Head or someone in a Leadership role is your loyal servant by the end of the day via the Persuasion Rack."
```